app/components/Footer.js

```diff
@@ -34,7 +34,7 @@ export default class Footer extends React.Component {
               <li><Scrollchor to="#schedule" animate={animeStyle}>Schedule</Scrollchor></li>
               <li><Scrollchor to="#faq" animate={animeStyle}>Faq</Scrollchor></li>
               <li><Scrollchor to="#sponsors" animate={animeStyle}>Sponsors</Scrollchor></li>
-              <li><span className="apply"><a href="https://uclafoundersschool.typeform.com/to/G7VxEU">Apply</a></span></li>
+              <li><span className="apply"><a href="https://foundersschool.typeform.com/to/aYuNYI">Apply</a></span></li>
```

Comment:
I think the idea is to add an apply button to the page like the site from last year, not to just add it to the navbar and footer

Reply:
do you want me to replace the "Join our mailing list" box?